frontend_build/src/prettier-frontend.js

```diff
@@ -34,24 +34,27 @@ function prettierFrontend({ file, write, encoding = 'utf-8', prettierOptions })
       );
       try {
         // Raw JS or SCSS file
-        if (file.endsWith('.js') || file.endsWith('.scss')) {
+        if (file.endsWith('.js')) {
+          options.parser = 'babylon';
```

Comment:
Is this just to be explicit (this is the prettier default)?

Reply:
I think that I was getting a warning that in the future this would be a required option

Reply:
(or maybe it was just being explicit, since I started re-using the `options` object throughout the function)